app/src/main/java/org/p2p/wallet/auth/ui/restore/common/CommonRestoreFragment.kt

```diff
@@ -122,6 +124,20 @@ class CommonRestoreFragment :
         }
     }
 
+    override fun setGoogleFlowEnabled(isEnabled: Boolean) {
```

Comment:
we can't disable the google flow itself. 
but we can hide RestoreViaGoogleFlow on the screen